### ISSUE TITLE
High contrast blocks

### ISSUE THREAD

**Issue by samelhusseini** [Microsoft/pxt-chibitronics#164]
As per design meeting, update the high contrast theme to show an empty black background of blocks and increase stroke size, such that: 

<img width="837" alt="screen shot 2018-01-10 at 3 03 57 pm" src="https://user-images.githubusercontent.com/16690124/34800187-99fe1a60-f617-11e7-8a6b-557bd01dd044.png">
